include/bus_handler.hpp

```diff
@@ -116,6 +121,18 @@ class BusHandler
      * @param[in] event: Button event
      */
     void btnRequest(int event);
+
+    /**
+     * @brief API to execute panel function number.
+     *
+     * Method to trigger panel function via d-bus. Currently only certain
+     * functions are supported to trigger using d-bus calls.
```

Comment:
Remove Currently only certain
      * functions are supported to trigger using d-bus calls.

Reply:
done